container/Containerfile

```diff
@@ -19,5 +19,6 @@ COPY conf.d /etc/ovirt-imageio/conf.d
 COPY file.json file.json
 COPY disk.img /images
 RUN chown ovirtimg:ovirtimg /images/disk.img
+VOLUME [ "/pki/system" ]
```

Comment:
Where this directory comes from?

I think we can ignore tls for now, better leave it to next step. It will be good enough to have a working container supporting only http as first step.

In k8s the container will likely use http, and https will be handled by ingress (e.g. nginx).

Reply:
If I understood correctly the docs, this creates a mount point at `/pki/system`, so that it can be mounted at runtime by doing `podman run -v /source/folder:/pki/system`. Alternatively, `CMD --mount` (bind), would require setting a source folder at build time, which is more inconvenient, unless we obtain it with an env variable (maybe).

So I opted for the `VOLUME` solution. But I am exploring possibilities, and any feedback is welcome.

Reply:
Not sure what VOLUME gives us - if you use:

    podman run -v /source/folder:/pki/system

It works without specifying VOLUME in the container file.

Reply:
I was not sure, I thought it was just to set an expected mount point inside the container. Actually, declaring a VOLUME in a containerfile creates an anonymous persistent volume that can be found with `podman volume ls` if you do not specify a runtime volume. This can be later shared between containers if you use `--volume-from`.

I think it may not be interesting for our usecase. In fact, I have found this article that discourages the use of volumes in dockerfiles: https://boxboat.com/2017/01/23/volumes-and-dockerfiles-dont-mix/